docs/reference/patch_acceptance_criteria.rst

```diff
@@ -0,0 +1,297 @@
+
+Example::
+
+    Subject: [SRU][O/N][PATCH v2 0/1] ALSA: hda/realtek: fix mute/micmute LEDs for a HP EliteBook 645 G10
+
+    BugLink: https://bugs.launchpad.net/bugs/2087983
+
+    ...
+
+The link must not be in its long form
+``https://bugs.launchpad.net/ubuntu/+source/linux/+bug/XXXXXX``.
+
+*Bad* example::
+
+    Subject: [SRU][O/N][PATCH v2 0/1] ALSA: hda/realtek: fix mute/micmute LEDs for a HP EliteBook 645 G10
+
+    BugLink: https://bugs.launchpad.net/ubuntu/+source/linux/+bug/2087983
+
+    ...
+
+SRU cover letter
+^^^^^^^^^^^^^^^^
+
+The patch must come with a *cover letter* that has both a short link to the SRU
+bug and a copy of the *SRU Justification* from the bug. It can be generated
+using both the ``--cover-letter`` and the ``--compose`` options of the
+git-cherry-pick_ command.
+
+Example cover letter::
+
+    Subject: [SRU][O/N][PATCH v2 0/1] ALSA: hda/realtek: fix mute/micmute LEDs for a HP EliteBook 645 G10
+
+    BugLink: https://bugs.launchpad.net/bugs/2087983
+
+    SRU Justification:
+
+    [ Impact ]
+
+    Mute/mic LEDs don't function on HP EliteBook 645 G10.
+
+    [ Test Plan ]
+
+    Test mute and mic LEDs with proposed kernel once patched.
+
+    [ Where problems could occur ]
+
+    Unknown regressions in the sound subsystem.
+
+    Maksym Glubokiy (1):
+      ALSA: hda/realtek: fix mute/micmute LEDs for a HP EliteBook 645 G10
+
+     sound/pci/hda/patch_realtek.c | 1 +
+     1 file changed, 1 insertion(+)
+
+Cherry-pick or backport
+-----------------------
+
+This section describes additional reasons why the Ubuntu Kernel Team won't
+approve a cherry-pick or backport patch.
+
+Upstream
+^^^^^^^^
+
+The patch should be in the mainline_ or the stable_ tree. Having the patch in
+a maintainer subtree is not enough, because the subtree might change. Having
+the patch in linux-next_ is bare minimum.
+
+.. _mainline: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/
+.. _stable: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/
+.. _linux-next: https://www.kernel.org/doc/man-pages/linux-next.html
+
+Source
+^^^^^^
+
+.. highlight:: text
+
+The patches should have a *cherry picked from* or *backported from* line with
+the appropriate sha from the upstream. It can be generated using the
+``-x`` option of the git-cherry-pick_ command. This line should appear just
+before your *Signed-off-by*::
+
+    (cherry picked from commit 622f21994506e1dac7b8e4e362c8951426e032c5)
+
+::
+
+    (backported from commit 622f21994506e1dac7b8e4e362c8951426e032c5)
+
+In case the upstream source is linux-next, you should explicit it::
+
+    (cherry picked from commit 622f21994506e1dac7b8e4e362c8951426e032c5 linux-next)
+
+In case the provenance is anything else, you should explicit the source git
+tree in full::
+
+    (cherry picked from commit 622f21994506e1dac7b8e4e362c8951426e032c5 git://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git)
```

Comment:
Though we say before that non-upstream sources will rather be frowned upon. IMO not a blocker. Just mentioning it. Thinking about it maybe one other special case might be added which has happened recently. Picking or backporting from another series (which might even be SAUCE there).